core-bundle/src/EventListener/PreviewUrlConvertListener.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener;
+
+use Contao\ArticleModel;
+use Contao\CoreBundle\Event\PreviewUrlConvertEvent;
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\PageModel;
+
+class PreviewUrlConvertListener
+{
+    private $framework;
+
+    public function __construct(ContaoFramework $framework)
+    {
+        $this->framework = $framework;
+    }
+
+    public function onPreviewUrlConvert(PreviewUrlConvertEvent $event): void
```

Comment:
The method should be changed to `__invoke()` so we can simplify the event registration (see #1125).